### ISSUE TITLE
Prometheus module does not handle spaces in label values

### ISSUE THREAD

**Issue by yolo-wang** [netdata/go.d.plugin#536]
The Prometheus metrics collector does not appear to handle spaces in the label values. For example, this metric:
```
jvm_memory_committed_bytes{area="nonheap",id="Code Cache",} 6.1997056E7
jvm_memory_committed_bytes{area="nonheap",id="Compressed Class Space",} 1.1927552E7
jvm_memory_committed_bytes{area="heap",id="Eden Space",} 2.86326784E8
jvm_memory_committed_bytes{area="heap",id="Survivor Space",} 3.5782656E7
jvm_memory_committed_bytes{area="heap",id="Tenured Gen",} 7.15849728E8
```

Will cause the following exception:
```
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_gc_pause_seconds_count' : unacceptable symbol in dim ID 'jvm_gc_pause_seconds_count|action=end of minor GC,cause=Allocation Failure' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_gc_pause_seconds_count' : unacceptable symbol in dim ID 'jvm_gc_pause_seconds_count|action=end of major GC,cause=Metadata GC Threshold' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_gc_pause_seconds_max' : unacceptable symbol in dim ID 'jvm_gc_pause_seconds_max|action=end of minor GC,cause=Allocation Failure' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_gc_pause_seconds_max' : unacceptable symbol in dim ID 'jvm_gc_pause_seconds_max|action=end of major GC,cause=Metadata GC Threshold' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_gc_pause_seconds_sum' : unacceptable symbol in dim ID 'jvm_gc_pause_seconds_sum|action=end of major GC,cause=Metadata GC Threshold' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_gc_pause_seconds_sum' : unacceptable symbol in dim ID 'jvm_gc_pause_seconds_sum|action=end of minor GC,cause=Allocation Failure' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_memory_committed_bytes' : unacceptable symbol in dim ID 'jvm_memory_committed_bytes|area=nonheap,id=Compressed Class Space' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_memory_committed_bytes' : unacceptable symbol in dim ID 'jvm_memory_committed_bytes|area=heap,id=Tenured Gen' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_memory_committed_bytes' : unacceptable symbol in dim ID 'jvm_memory_committed_bytes|area=heap,id=Eden Space' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_memory_committed_bytes' : unacceptable symbol in dim ID 'jvm_memory_committed_bytes|area=nonheap,id=Code Cache' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_memory_committed_bytes' : unacceptable symbol in dim ID 'jvm_memory_committed_bytes|area=heap,id=Survivor Space' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_memory_max_bytes' : unacceptable symbol in dim ID 'jvm_memory_max_bytes|area=heap,id=Survivor Space' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_memory_max_bytes' : unacceptable symbol in dim ID 'jvm_memory_max_bytes|area=nonheap,id=Compressed Class Space' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_memory_max_bytes' : unacceptable symbol in dim ID 'jvm_memory_max_bytes|area=heap,id=Tenured Gen' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_memory_max_bytes' : unacceptable symbol in dim ID 'jvm_memory_max_bytes|area=heap,id=Eden Space' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_memory_max_bytes' : unacceptable symbol in dim ID 'jvm_memory_max_bytes|area=nonheap,id=Code Cache' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_memory_used_bytes' : unacceptable symbol in dim ID 'jvm_memory_used_bytes|area=heap,id=Tenured Gen' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_memory_used_bytes' : unacceptable symbol in dim ID 'jvm_memory_used_bytes|area=nonheap,id=Code Cache' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_memory_used_bytes' : unacceptable symbol in dim ID 'jvm_memory_used_bytes|area=heap,id=Eden Space' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_memory_used_bytes' : unacceptable symbol in dim ID 'jvm_memory_used_bytes|area=heap,id=Survivor Space' : ' '
[ WARN  ] prometheus[prometheus] collect_gauge_counter.go:41 error on adding dim to chart 'jvm_memory_used_bytes' : unacceptable symbol in dim ID 'jvm_memory_used_bytes|area=nonheap,id=Compressed Class Space' : ' '
```

**Comment by pickfire** [comment]
```
2021-01-08 07:58:42: go.d WARNING: prometheus[cadvisor-monitoring_cadvisor-zt4qn_cadvisor_tcp_8080] error on adding dim to chart 'cadvisor_version_info' : unacceptable symbol in dim ID 'cadvisor_version_info|cadvisorRevision=de723a09,cadvisorVersion=v0.30.2,dockerVersion=20.10.0,kernelVersion=4.19.157,osVersion=Alpine Linux v3.7' : ' '
```

Same issue for me. It happened to cadvisor.

**Comment by ilyam8** [comment]
Thx for reporting the bug, guys. The fix is #537.
